.github/actions/build-image/action.yaml

```diff
@@ -0,0 +1,31 @@
+name: Build container image
+description: Build container image and check vulnerabilities
+
+inputs:
+  image_tag:
+    description: The tag of the image
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
+
+    - name: Build container image
+      uses: docker/build-push-action@2eb1c1961a95fc15694676618e422e8ba1d63825
+      with:
+        tags: ${{ env.image_tag }}
```

Comment:
Just a curiosity: is there any difference to access the `image_tag` property using `env.image_tag` or `inputs.image_tag`?

Reply:
Honestly I do not know. It should be `inputs.image_tag`. I'm going to change it.